tee-worker/enclave-runtime/src/rpc/worker_api_direct.rs

```diff
@@ -199,6 +202,36 @@ where
 		}
 	});
 
+	if cfg!(not(feature = "production")) {
+		// mrenclave_updateScheduled
+		// params: sidechainBlockNumber, hex encoded mrenclave
+		let mrenclave_update_scheduled_name: &str = "mrenclave_updateScheduled";
+		io.add_sync_method(mrenclave_update_scheduled_name, move |params: Params| {
+			match params.parse::<(SidechainBlockNumber, String)>() {
+				Ok((bn, mrenclave)) =>
+					return match hex::decode(&mrenclave) {
+						Ok(mrenclave) => {
+							let mut enclave_to_set: MrEnclave = [0u8; 32];
+							enclave_to_set.copy_from_slice(&mrenclave);
+							return match GLOBAL_SCHEDULED_ENCLAVE.update(bn, enclave_to_set) {
+								Ok(()) => Ok(json!("{}")),
```

Comment:
I'm not sure this call should return value and why only `new_mrenclave` instead of tuple or even whole map.
But yes, in general I think I need to change the response.

Reply:
`RpcReturnValue` only has a `value` field for hex string, so we need encoded tuple/map if we want, I was just lazy and use mrenclave as the value, similar to substrate event :(